Source/WebCore/Modules/permissions/Permissions.h

```diff
@@ -49,6 +51,7 @@ class Permissions : public RefCounted<Permissions> {
 
     NavigatorBase* navigator();
     void query(JSC::Strong<JSC::JSObject>, DOMPromiseDeferred<IDLInterface<PermissionStatus>>&&);
+    static std::optional<PermissionQuerySource> sourceFromContext(const ScriptExecutionContext&);
```

Comment:
No need to declare in the header since it is not used from outside this class.

Reply:
Fixed!